components/ItemCripto/ItemCripto.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+
+import { ItemCriptoProps } from '../../utils/interfaces';
+
+import {
+  CriptoContainer,
+  SeconContainer,
+  CriptoName,
+  CriptoPercentageGreen,
+  CriptoPercentageRed,
+} from './ItemCripto-Styles';
+
+const ItemCripto = ({item}: ItemCriptoProps) => {
+
+  let auxPercentage = item.percentage.toString();
+
+  if (auxPercentage.substring(0, 1) === '-') {
```

Comment:
unnecessary check this doc, and update the validations and use of the item.percentage: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs